server/template.go

```diff
@@ -2,13 +2,17 @@ package main
 
 import (
 	"bytes"
+	"regexp"
 	"strings"
 	"text/template"
 
 	"github.com/Masterminds/sprig/v3"
 	"github.com/pkg/errors"
 )
 
+const mdCommentRegexPattern string = `(<!--.*?-->)|(<!--[\S\s]+?-->)`
```

Comment:
@M-ZubairAhmed, can you elaborate on the need to match both `.*?` and `[\S\s]+?`?  I don't see any test cases that seem to require one over the other, so I'm wondering if there's an opportunity to converge on the latter condition instead.

Reply:
Hi, its an or condition not an and one. The first one checks for inline or single line comments and the second one checks for multiline comments. I think we have both scenarios in the app. Do you want me to add multiline scenario in the tests ?

Reply:
Here is the regex case 
https://regex101.com/r/az8Lu6/3

Reply:
Thanks for the clarification, @M-ZubairAhmed! Yes, let's definitely add newline test coverage regardless.

I note that, using regex101.com, that `(<!--[\S\s]+?-->)` alone still matches both cases. Do we need to match on `.*?` also?

Reply:
Let me run and update tests and get back to you with an answer on that.